src/Android/Xamarin.Android/Samples/Geometry/FormatCoordinates/FormatCoordinates.cs

```diff
@@ -0,0 +1,201 @@
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Drawing;
+
+namespace ArcGISRuntimeXamarin.Samples.FormatCoordinates
+{
+    [Activity]
+    public class FormatCoordinates : Activity
+    {
+        // Create and hold reference to the used MapView
+        private MapView _myMapView = new MapView();
+        private EditText _DecimalDegreesEditText;
+        private EditText _DmsEditText;
+        private EditText _UtmEditText;
+        private EditText _UsngEditText;
+
+        // Hold last edited field
+        private EditText _lastEdited;
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Format coordinates";
+
+            // Create the UI, setup the control references and execute initialization
+            CreateLayout();
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Create the map
+            _myMapView.Map = new Map(Basemap.CreateStreets());
+
+            // Add the graphics overlay to the map
+            _myMapView.GraphicsOverlays.Add(new GraphicsOverlay());
+
+            // Create the starting point
+            MapPoint startingPoint = new MapPoint(0, 0, SpatialReferences.WebMercator);
+
+            // Update the UI with the initial point
+            UpdateUiFromMapPoint(startingPoint);
+
+            // Subscribe to map tap events to enable tapping on map to update coordinates
+            _myMapView.GeoViewTapped += (sender, args) => { UpdateUiFromMapPoint(args.Location); };
+        }
+
+        private void InputTextChanged(object sender, EventArgs e)
+        {
+            // Keep track of the last edited field
+            _lastEdited = (EditText)sender;
+        }
+
+        private void ProcessTextChange(object sender, EventArgs e)
+        {
+            // Hold the entered point
+            MapPoint enteredPoint = null;
+
+            // Update the point based on which textbox sent the event
+            try
+            {
+                switch (_lastEdited.Hint)
+                {
+                    case "Decimal Degrees":
+                    case "Degrees, Minutes, Seconds":
+                        enteredPoint =
+                            CoordinateFormatter.FromLatitudeLongitude(_lastEdited.Text, _myMapView.SpatialReference);
+                        break;
+
+                    case "UTM":
+                        enteredPoint =
+                            CoordinateFormatter.FromUtm(_lastEdited.Text, _myMapView.SpatialReference, UtmConversionMode.LatitudeBandIndicators);
```

Comment:
I would suggest using `UtmConversionMode.NorthSouthIndicators` instead of `UtmConversionMode.LatitudeBandIndicators`. I think the N/S zone indicator is the more familiar way (for most folks) of viewing these coordinates.